### PR TITLE
Answers

### PR DESCRIPTION
**Pull request by stiehlrod**
- new table and model
- new model and request specs
- new serializer and endpoint

* pivoted based on meeting with a teammate from working on a union to creating 2 endpoints, one for responses and one for answers.

### CODE REVIEW

File: app/models/response.rb [hunk 1]
```diff
@@ -1,17 +1,32 @@
 class Response < ActiveRecord::Base
   belongs_to :choice
-  delegate :question, :to => :choice
+  has_many :answers
+  delegate :question, :to => :choice, :allow_nil => true
+  # delegate :question, :to => :answer
```
Comment (blake-enyart) [inline app/models/response.rb]: Do you need this commented section for reference later?

File: spec/requests/api/v1/q_and_a_spec.rb [hunk 2]
```diff
@@ -6,8 +6,8 @@
     q2 = c1.questions.create(question: "How much sleep did you get?", created_at: "2019-05-23 13:31:38 UTC", updated_at: "2019-05-23 13:31:38 UTC")
     yes = q1.choices.create(choice:"yes", correct:nil)
     no = q1.choices.create(choice:"no", correct:nil)
-    yes_2 = q2.choices.create(choice:"yes", correct:nil)
-    no_2 = q2.choices.create(choice:"no", correct:nil)
+    yes_2 = q2.choices.create(choice:"usual", correct:nil)
```
Comment (blake-enyart) [inline spec/requests/api/v1/q_and_a_spec.rb]: What do these variable names mean?

**Review by blake-enyart — COMMENTED** [review]
Other than the commented out code which I marked and the question I have about the names in the testing suite I think this looks great and can be merged in after that.

**Comment by carriewalsh** [comment]
Yay CSS!

**Pull request merged**